apps/hash-api/src/graph/ontology/primitive/data-type.ts

```diff
@@ -120,6 +122,48 @@ export const getDataTypeById: ImpureGraphFunction<
   return dataType;
 };
 
+/**
+ * Get an data type rooted subgraph by its versioned URL and if it's not available in the Graph and is an external type it will load this type into the Graph.
```

Comment:
```suggestion
 * Get a data type rooted subgraph by its versioned URL.
 *
 * If the type does not already exist within the Graph, and is an externally-hosted type, this will also load the type into the Graph.
```

Perhaps something like this for the docs on these